src/main/java/org/broadinstitute/consent/http/util/gson/DateTypeAdapter.java

```diff
@@ -24,8 +24,12 @@ public Date deserialize(JsonElement json, Type type, JsonDeserializationContext
       throws JsonParseException {
     try {
       return new Date(json.getAsLong());
-    } catch (DateTimeParseException e) {
-      throw new JsonParseException(e.getMessage());
+    } catch (NumberFormatException e) {
+      try {
+        return new Date(json.getAsString());
```

Comment:
`Date(String s)` is deprecated. It looks like this will work in tests:
```
      try {
        return new SimpleDateFormat("LLL dd, yyyy, hh:mm:ss a").parse(json.getAsString());
      } catch (ParseException e1) {
        throw new JsonParseException(e1.getMessage());
      }
```